docs/integrations/github/discovery.md

```diff
@@ -163,6 +163,13 @@ catalog:
           branch: 'main' # string
           repository: '.*' # Regex
         validateLocationsExist: true # optional boolean
+      visibilityProviderId:
+        organization: 'backstage' # string
+        catalogPath: '/catalog-info.yaml' # string
+        filters:
+          visibilities:
```

Comment:
please add this new filter to the configuration options below the code snippet, too.

Reply:
Unresolved this one since it's not handled, the idea is to add this to the markdown docs below too

Reply:
oh ok, it misunderstood the request. I'm onit